main.cpp

```diff
@@ -131,7 +131,14 @@ tapeMachine.play();
 tapeMachine.record();
 //  action 3:  the tape machine can be set to always rewind to a position
 tapeMachine.rewindToPosition(); 
-
+//  11)
+//  noun:saladShooter
+//  action 1: the salad shooter slices cucumbers
+saladShooter.sliceCucumber();
+//  action 2: the salad shooter dices carrots
+saladShooter.diceCarrots();
+//  action 3: the salad shooter self cleans
+saladShooter.selfClean();
```

Comment:
Good, but I needed you to replace the one I marked with this one.  
- [ ] revise so I know you're comfortable with the commit/pull request/review process.